core/kits/documents/tabs/settings-layout.php

```diff
@@ -34,7 +36,9 @@ public function get_help_url() {
 	}
 
 	protected function register_tab_controls() {
-		$default_breakpoints = Responsive::get_default_breakpoints();
+		$default_breakpoints = Breakpoints_Manager::get_default_config();
```

Comment:
Add `default_value` to the returned `Breakpoint` config and use it instead of `get_default_config`

Reply:
Called `Breakpoint::get_default_value()` instead. Done.